test/conformance/gateway_conformance_test.go

```diff
@@ -50,5 +55,28 @@ func TestGatewayConformance(t *testing.T) {
 		BaseManifests:    conformanceTestsBaseManifests,
 	})
 	cSuite.Setup(t)
-	// TODO: cSuite.Run(t, tests.ConformanceTests)
+
+	t.Log("configuring gateway conformance tests")
+	tts := make([]suite.ConformanceTest, 0, len(tests.ConformanceTests))
+	for _, tt := range tests.ConformanceTests {
+		ttcpy := tt
+		for i, manifest := range ttcpy.Manifests {
+			ttcpy.Manifests[i] = fmt.Sprintf("%s/conformance/%s", manifestRepo, manifest)
+		}
+		tts = append(tts, ttcpy)
+	}
+
+	t.Log("running gateway conformance tests")
+	for _, tt := range tts {
+		if enabledGatewayConformanceTests.Has(tt.ShortName) {
+			t.Run(tt.Description, func(t *testing.T) { tt.Run(t, cSuite) })
+		}
+	}
 }
+
+// TODO: this is a temporary list to filter out tests that we're not
+// ready to run yet. We will eventually run all the tests and that work
+// is being tracked in https://github.com/Kong/kubernetes-ingress-controller/issues/2210
```

Comment:
suggestion to make this TODO less wishful :)

```suggestion
// Today we run only the subset below of all Gateway conformance tests.
// TODO: ensure that this module runs all Gateway conformance tests
// https://github.com/Kong/kubernetes-ingress-controller/issues/2210
```